app/controllers/discovered_hosts_controller.rb

```diff
@@ -29,6 +30,11 @@ def show
 
     # summary report text
     @report_summary = nil
+    init_regex_and_categories
+    @host.facts_hash.each do |key, value|
+      value = number_to_human_size(value)  if /size$/.match(key)
```

Comment:
Double space before `if`.

Can you also proceed with expanding Settings chapter in our manual? http://theforeman.org/plugins/foreman_discovery/4.0/index.html#3.1.4Globalsettings it is this page: https://github.com/theforeman/theforeman.org/blob/gh-pages/plugins/foreman_discovery/4.0/index.md